db/migrate/20211202134547_migrate_office365_user_info.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+class MigrateOffice365UserInfo < ActiveRecord::Migration[6.1]
+  def up
+    execute <<~SQL
+    INSERT INTO user_associated_accounts (
+      provider_name,
+      provider_uid,
+      user_id,
+      info,
+      last_used,
+      created_at,
+      updated_at
+    ) SELECT
+      'microsoft_office365',
```

Comment:
`microsoft_office365`?

Reply:
As above, we can't change it without a lot of pain